packages/cli-lib/lang/en.lyaml

```diff
@@ -564,6 +565,24 @@ en:
                 describe: "Name to use for created sandbox"
             success:
               create: "Sandbox \"{{ name }}\" with portalId \"{{ sandboxHubId }}\" created successfully."
+              prePrompt: "
+
+
+              The following step will prompt you to authenticate with sandbox \"{{ name }}\" and add it to the CLI config.
+
+              "
+              configFileUpdated: "{{ configFilename }} updated with {{ authMethod }} for account {{ account }}."
+            failure:
+              scopes: "
```

Comment:
Not sure the best way to do this, but I added a bunch of white space here so the messaging wouldn't get bunched up/all on one line

<img width="1036" alt="Screen Shot 2022-05-06 at 4 22 12 PM" src="https://user-images.githubusercontent.com/16788677/167211375-c4d01ad5-47d3-4d67-aa58-cb10574697a8.png">

Reply:
Nice 👍  I like it with the extra spacing. FWIW I've been using blank `logger.log` calls to add newlines. You can also use `\n` in the en.lyaml file to add newlines (there should be examples of it in there somewhere).